src/main/kotlin/me/odinmain/features/impl/nether/NoPre.kt

```diff
@@ -81,62 +82,172 @@ object NoPre : Module(
         return when (missing) {
             // Shop Missing
             SupplyPickUpSpot.Shop -> when (preSpot) {
-                SupplyPickUpSpot.Triangle, SupplyPickUpSpot.X -> "Go X Cannon"
-                SupplyPickUpSpot.Equals, SupplyPickUpSpot.Slash -> "Go Square, place on Shop"
-                else -> ""
+                SupplyPickUpSpot.Triangle, SupplyPickUpSpot.X -> {
```

Comment:
Make it return a pair for cleaner syntax
```suggestion
                SupplyPickUpSpot.Triangle, SupplyPickUpSpot.X ->  SupplyPickUpSpot.xCannon to "Go X Cannon
```

Reply:
committed